docs/Automation.md

```diff
@@ -20,164 +15,103 @@ and is not expected to change at any future time. If you have a scenario that
 
-#### Defined Constants
+The **Errors** property contains **ScanResult** objects which are the result of a single rule test on a single element and have the following properties:
 
-The following constants are defined in the assembly:
-
-**Name** | **Value** | **Description**
+**Name** | **Type** | **Description**
 ---|---|---
-CommandConstStrings.TargetProcessId | “TargetProcessId” | The string representation (in decimal format) of the process ID to be scanned. Must be set in at least one tier (Tier 1/Tier 2/Tier 3).
-CommandConstStrings.OutputPath | “OutputPath” | The file location where files from this automation session are to be written. If the target path directory does not exist, it will be created when the first output file is written to it. If the intent is for CI / CD, it is recommended that OutputPath not be set.
-CommandConstStrings.OutputFile | “OutputFile” | The specific file (without path) where scan results are to be written. If the file already exists, it will be overwritten. The file extension can (and should) be omitted. Must be set in at least one tier (Tier 1/Tier 2/Tier 3).
-CommandConstStrings.OutputFileFormat | “OutputFileFormat” | The specific format that the output file should be generated in. The files generated will use the output path and the output filename provided. The three supported values are:<ul><li> **a11yTest** will generate "\*.a11ytest" files.<li>**Sarif** = will generate "\*.sarif" files.<li>**All** will generate both an a11ytest file and a sarif file.</ul>This parameter has lower precedence than if an extension was passed in as a part of the filename. If no extension is passed in the filename and this parameter is not passed in, then files will be generated in the sarif format. May be set in at least one tier (Tier 1/Tier 2/Tier 3). *Default value = **Sarif***.
-CommandConstStrings.NoViolationPolicy | “NoViolationPolicy” | Specifies how output files should be handled if no violations are found. <ul><li>**Discard** will discard the results file.<li>**Retain** will retain the results file.</ul>*Default value = **Retain***. Valid only as a Tier 3 setting.
-CommandConstStrings.Discard  | “Discard” | Value for NoViolationPolicy.
-CommandConstStrings.Retain | “Retain” | Value for NoViolationPolicy.
-CommandConstStrings.TeamName | “TeamName” | Value for Team name in telemetry.
+Rule | `RuleInfo` | Information about the rule (description, how to fix information, etc.) that was evaluated on the element.
+Element | `ElementInfo` | The element which was tested against the rule.
 
 ### Using the assembly
 You can get the files via a NuGet package Configure NuGet to retrieve the
 **Axe.Windows** package from
 <https://api.nuget.org/v3/index.json>,
 then use the classes in the Axe.Windows.Automation namespace (see
-examples below):
+example below):
 
-#### From .NET code
 -   Prerequisite: Your project *must* use .NET 4.7.1 (this is required by
```

Comment:
>4.7.1 [](start = 47, length = 5)

Long-term, we might want to consider allowing earlier versions (the 4.7.1 is to get a fix that AIWin needed)

Reply:
Perhaps, but let's leave it until there's a compelling reason to downgrade.